docs/take-to-prod.md

```diff
@@ -0,0 +1,108 @@
+Taking your Spark .Net Application to Production
+===
+
+# Table of Contents
+This how-to provides general instructions on how to take your .NET for Apache Spark application to production.
+In this documentation, we will summary the most commonly asked scenarios when running Spark .Net Application.
+And you will also learn how to package your application and submit your application with [spark-submit](https://spark.apache.org/docs/latest/submitting-applications.html) and [Apachy Livy](https://livy.incubator.apache.org/).
+- [How to take your application to production when you have single dependency](#how-to-take-your-application-to-production-when-you-have-single-dependency)
+  - [Scenarios - Scenario 1 and Scenario 2](#scenarios---single-dependency)
+  - [Package your application](#package-your-application---single-dependency)
+  - [Launch your application](#launch-your-application---single-dependency)
+- [How to take your application to production when you have multiple dependencies](#how-to-take-your-application-to-production-when-you-have-multiple-dependencies)
+  - [Scenarios - Scenario 3, Scenario 4, Scenario 5 and Scenario 6](#scenarios---multiple-dependencies)
+  - [Package your application](#package-your-application---multiple-dependencies)
+  - [Launch your application](#launch-your-application---multiple-dependencies)
+
+## How to take your application to production when you have single dependency
+### Scenarios - single dependency
+#### Scenario 1. SparkSession code and business logic in the same Program.cs file
+This would be the simple usecase when you have SparkSession code and business logic (UDFs) in the same Program.cs file and in the same project (e.g. mySparkApp.csproj).
+#### Scenario 2. SparkSession code and business logic in the same project, but different .cs files
+This would be the usecase when you have SparkSession code and business logic (UDFs) in the different .cs files but in the same project (e.g. SparkSession in Program.cs, business logic in BusinessLogic.cs and both are in mySparkApp.csproj).
```

Comment:
```suggestion
This would be the use case when you have `SparkSession` code and business logic (UDFs) in the different .cs files but in the same project (e.g. `SparkSession` in Program.cs, business logic in BusinessLogic.cs and both are in mySparkApp.csproj).
```